grammar.js

```diff
@@ -143,7 +143,10 @@ module.exports = grammar({
 
     _decorated_statement: $ => seq(
       repeat1($.decorator),
-      $.declaration,
+      choice(
+        $.declaration,
+        $.call_expression
```

Comment:
Maybe we should use `expression_statement` here. Since decorators/attributes can appear anywhere.